src/settings.rs

```diff
@@ -81,6 +81,9 @@ pub struct Settings {
     pub adm_ignore_advertisers: Option<String>,
     /// a JSON list of advertisers to allow for versions of firefox less than 91.
     pub adm_has_legacy_image: Option<String>,
+    /// a JSON list of location DMAs to never return (population less than 15K)
+    pub exclude_dma: Option<String>,
+    pub excluded_dma: Option<Vec<u16>>,
```

Comment:
I think this should live in `ServerState` instead, otherwise this adds an ignored `CONTILE_EXCLUDED_DMA` setting that could be confused for the real setting later.

It'd just need the deserializing moved into `Server::with_settings`